applications/nrf_desktop/README.rst

```diff
@@ -798,6 +798,27 @@ By default, this period is set to 120 seconds.
     If a nRF Desktop device supports remote wakeup, the USB connected device goes to suspended state when USB is suspended.
     The device can then trigger remote wakeup of the connected host on user input.
 
+Configuration
+*************
+
+|config|
+
+Adding nRF21540 EK shield support
+=================================
+
+The nRF Desktop application can be used with the :ref:`ug_radio_fem_nrf21540_ek` shield, an RF front-end module (FEM) for the 2.4 GHz range extension.
+Any nRF Desktop HID application can be built with the shield support by passing the ``SHIELD`` parameter to the build command.
+
+For detailed information about building the nRF Desktop application for the nRF21540 EK, see the :ref:`ug_radio_fem_nrf21540_ek_programming` section on the nRF21540 EK documentation page.
+
+.. note::
+   For the multi-core build, use the ``hci_rpmsg_`` parameter.
```

Comment:
parameter -> child image name ?
Maybe even: `use the ``hci_rpmsg_`` child image name, because in nRF Desktop network core runs ``hci_rpmsg``

Reply:
Sounds good to me.

Reply:
Done